test/one/profiler/test/Runner.java

```diff
@@ -139,7 +139,10 @@ private static boolean run(Method m) {
                 continue;
             }
 
-            log.log(Level.INFO, "Running " + testName + "...");
+            log.log(Level.INFO, "Running " + testName +
+                (test.args() != null && test.args().length() > 0 ? " args: " + test.args() : "" ) +
+                (test.inputs() != null && test.inputs().length > 0 ? " inputs: [" + String.join(" ", test.inputs()) + "]": "" ) +
```

Comment:
Nit: Maybe "with inputs" instead of just "inputs"?

Reply:
Then we may need `with args` too. Given that this is printed only during running tests, I would keep them brief.